docs/content/querying/dimensionspecs.md

```diff
@@ -263,13 +263,18 @@ Note, if you are working with the `__time` dimension, you should consider using
 [time extraction function instead](#time-format-extraction-function) instead,
 which works on time value directly as opposed to string values.
 
-Time formats are described in the
-[SimpleDateFormat documentation](http://icu-project.org/apiref/icu4j/com/ibm/icu/text/SimpleDateFormat.html)
+If "joda" is true, time formats are described in the [Joda DateTimeFormat documentation](http://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html).
+If "joda" is false (or unspecified) then formats are described in the [SimpleDateFormat documentation](http://icu-project.org/apiref/icu4j/com/ibm/icu/text/SimpleDateFormat.html).
```

Comment:
IMO, it would make more sense to make joda one default.
probably can be this in next major release.